coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/hierarchy/ProposalHierarchyServiceImpl.java

```diff
@@ -331,7 +331,6 @@ public DevelopmentProposal removeFromHierarchy(DevelopmentProposal childProposal
             hierarchyProposal.setHierarchyOriginatingChildProposalNumber(lowestProposalNumber);
             dataObjectService.save(childProposal);
             dataObjectService.save(hierarchyProposal);
-            synchronizeAllChildren(hierarchyProposal);
             return childProposal;
```

Comment:
One thing I want to point on in this PR @blackcathacker is I removed syncAll when unlinking from hierarchy since I do not think it is needed at this stage and could make this slow when all someone is trying to do is remove a proposal from hierarchy. If you think we need to keep this in for some reason, let me know and I will add it in and test it again. Does budget unlinking depend on this somehow?